geom/wkt_fuzz_test.go

```diff
@@ -0,0 +1,56 @@
+// +build gofuzzbeta
+
+package geom_test
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/peterstace/simplefeatures/geom"
+	"github.com/peterstace/simplefeatures/internal/extract"
+)
+
+func FuzzParseUnmarshalWKT(f *testing.F) {
```

Comment:
where is `testing.F` defined? I noticed there's a custom `gofuzzbeta` build tag here, maybe it's related somehow.

Reply:
Yes, that's right -- you need to use a custom branch of the Go toolchain to get this working (instructions [here](https://blog.golang.org/fuzz-beta)).